packages/skill-api/src/server/send-server-email.ts

```diff
@@ -20,22 +20,16 @@ function hashToken(token: string, options: any) {
   )
 }
 
-export async function sendServerEmail({
+export async function createVerificationUrl({
```

Comment:
Extracted the logic for building the verification URL from `sendServerEmail` into a `createVerificationUrl` function. That function then gets used by `sendServerEmail`, but can also be imported and used in the `createMagicLink` route.